### DISCUSSION TITLE
Repository/Project Selection

### DISCUSSION THREAD

**Discussion by mihir-bombay-studio** [dear-digital/linter#4] [Q&A]
There are several existing repositories that we can use as references or even build upon. 
Let's decide which ones would be most useful to us.
Make sure that repository has one of the free to use licenses.
Remember to share your reasoning why that project matches our scope!

**Comment by anishdalvi** [comment]
I found this [repo](https://github.com/mattzcarey/code-review-gpt.git) which reviews the code using Language Models of GPT. How about taking some reference from this?

**Reply by mihir-bombay-studio** [reply]
Nice repo 🙌 , I think we can start with a very basic linter from scratch otherwise it would be too difficult to read the project and then make changes for everyone. We are making a JSON prettifier and linter to kickoff and as the project evolves we'll add more functionalities to it. I'll add initialization steps/feature requests which anyone can submit PR for, first correct PR will be accepted.
